react-charts/bullet-chart/index.json

```diff
@@ -1,6 +1,6 @@
 {
-  "title": "Bullet chart",
-  "description": "Learn how to implement a React bullet chart.",
+  "title": "Bullet Chart",
```

Comment:
Use sentence case here and keep it to "Bullet chart"

Reply:
We don't use title case for titles?

Reply:
Fixed, just curious why this wouldn't be title case since it is a title.